### PR TITLE
Mockup for footer component

### PR DESCRIPTION
**Pull request by IVIURRAY**
Hey guys - made a quick mockup for the footer components...see below

image

### CODE REVIEW

File: frontend/src/pages/footer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import {
+    Grid,
+    Typography
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(theme => ({
+    root: {
+      flexGrow: 1,
+      backgroundColor: '#e0e0e0',
```
Comment (IVIURRAY) [inline frontend/src/pages/footer.js]: Can I get this from theme somewhere?

Comment (sean-clayton) [reply frontend/src/pages/footer.js]: Maybe try here? https://material-ui.com/system/palette/#api Not sure 🤔

Comment (sean-clayton) [reply frontend/src/pages/footer.js]: Actually @IVIURRAY I think this is what you want: https://material-ui.com/customization/color/#color-palette

Comment (sean-clayton) [reply frontend/src/pages/footer.js]:
```js
import grey from '@material-ui/core/colors/grey';

// ...
const useStyles = makeStyles(theme => ({
    root: {
      flexGrow: 1,
      backgroundColor: grey[300],
// ...
```

**Pull request merged**